### PR TITLE
[Term Entry] Swift Arrays .randomElement() #2561

### PR DESCRIPTION
**Pull request by bruce0221**
### Description

Create a term entry for Swfit Arrays `.randomElement()`, which is issue #2561 .

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swift/concepts/arrays/terms/randomElement/randomElement.md
Comment (caupolicandiaz) [inline content/swift/concepts/arrays/terms/randomElement/randomElement.md:3]:
```suggestion
Description: 'Returns a random element from an array.'
```

File: content/swift/concepts/arrays/terms/randomElement/randomElement.md
Comment (caupolicandiaz) [inline content/swift/concepts/arrays/terms/randomElement/randomElement.md:15]:
```suggestion
The ** `.randomElement()`** method will randomly return an element from an array. If the array is empty, it will return `nil`.
```

File: content/swift/concepts/arrays/terms/randomElement/randomElement.md
Comment (caupolicandiaz) [inline content/swift/concepts/arrays/terms/randomElement/randomElement.md:21]:
```suggestion
```

File: content/swift/concepts/arrays/terms/randomElement/randomElement.md
Comment (caupolicandiaz) [inline content/swift/concepts/arrays/terms/randomElement/randomElement.md:23]: Generally we try to include a sentence or two after the syntax block explaining the parameters, or functionality of the method, etc

File: content/swift/concepts/arrays/terms/randomElement/randomElement.md
Comment (caupolicandiaz) [inline content/swift/concepts/arrays/terms/randomElement/randomElement.md:34]:
```suggestion
The output will be similar to:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@bruce0221 I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by bruce0221** [comment]
@caupolicandiaz , the entry is revised.

**Review by caupolicandiaz — APPROVED** [review]
@bruce0221 I've gone ahead and made a few small edits. This is ready for a second review.

**Comment by bruce0221** [comment]
@caupolicandiaz  thank you. 
I need to wait for the second reviewer's approval, then I can merge the new branch into main branch, correct ?

**Comment by caupolicandiaz** [comment]
> @caupolicandiaz thank you.
> I need to wait for the second reviewer's approval, then I can merge the new branch into main branch, correct ?

Merging is done by the maintainers. The only work left, potentially, is addressing any comments in a second review.

**Review by SSwiniarski — APPROVED** [review]
@bruce0221 looks good to merge.

**Comment by SSwiniarski** [comment]
@bruce0221 Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/swift/arrays/randomElement

image

You're currently anonymous. If you don't wish to be anonymous, check out this forum thread on how to link your GitHub and Codecademy profiles: https://discuss.codecademy.com/t/my-contribution-is-live-but-im-not-being-shown-as-a-contributor-why-is-this/758036  **Note:** it may take a few minutes for the change to register on the entry page.

**Pull request merged**
